### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by buntin7419** [piejanssens/premiumizer#117]
It would be good if we could auto downloaded items that are downloaded via Premiumiz.me RSS feed.

i.e P.me auto downloads via it's RSS and Premiumizer then watches the folder and downloads to local.
Then ideally deletes the file a day later so stop P.me redownloading it.

Yay? or Nay?

**Comment by neox387** [comment]
I don't use the RSS feed feature, donno why you should except maybe for streaming directly in kodi without having to run any programs on a computer.. like sonarr/radarr ?

we can auto download everything that comes up in the downloader(transfer) list of premiumize.me with:
Download everything and assign the category 'default':

And why delete a day later ? if it's downloaded you have it ? ... :)

**Comment by buntin7419** [comment]
My reasoning is this.
P.me takes a custom RSS feed from the ShowRSS site. This means that those files are cached and 90% of the time instant.
So ultimately all new TV shows should in theory be instantly ready for download. Makes for a fast approch as P.me can can fail to download the torrent especially nzb's. 
The above approach will eliminate me manually having to download due to P.me failures.

Sonarr doesn't seem to retry failed downloads with Premiumizer like it does with Sabnzbd unless I'm mistaken?

**Comment by neox387** [comment]
Indeed it does not because it's Blackhole ni direct api integration, they are not in python etc to difficult for me xP

And why are they already cached? Because it takes longer for them to appear on showrrs then I Guess..

I use rarbg that's basicly allso rss feed with filters to only take rarbg rartv tags and amzn. Doesn't fail me xP

**Comment by neox387** [comment]
Not that simpel to just add the rss feed folder ^^

**Comment by buntin7419** [comment]
Not to worry. Just an idea to speed things up & Shave a couple of minutes of P.me download time. :)

They are already cached because many other people use the ShowRss feed to get their TV shows. Kodi users fill their 1TB quota full of TV for easy consumption using GAIA or the P.me addon.  P.me users also use it.
ShowRss only supply one or two links per quality so the odds are it's been requested many times previously before you request it. P.me as we know caches the links to prevent repeat downloads; so where as Rarg files usually download ShowRss files are usually already there.

**Comment by neox387** [comment]
If I have time i'll look into it,

So showrrs feed should be named tv for example so it downloads into a subfolder called tv so we know what category the download is...

And not sure about the add to transfer/download list option is that needed? I see if you delete it there it's also gone from my files.

Ye I don't see the appeal into streaming with kodi addons if PM or inet craps out you can't watch anything rather download ahead of watching.

**Comment by buntin7419** [comment]
The Rss feed can be named anything you want and is put in 'My Files/Feed Downloads' folder (My Files/Feed Downloads/WhateverYourFoderIsCalled)

My thought is as we can already download from a preconfigured tag, tv,movie etc. to a set location. Just add the ability to also add the RSS feed folder. I gave it a try the other day but it didn't download anything maybe due to the level the RSS folder is under. (not the root)

All guesswork, i'm no coder.

> And not sure about the add to transfer/download list option is that needed? I see if you delete it there it's also gone from my files.
Not sure what you mean by this.

> Ye I don't see the appeal into streaming with kodi addons if PM or inet craps out you can't watch anything rather download ahead of watching.
I agree. Doesn't hurt to get the Streamers to pre download my Shows though :D

**Comment by neox387** [comment]
Submit existing feed items to Downloader

Is an option when adding a feed, not sure about it's purpose

**Comment by neox387** [comment]
RSS support working also, working on delay time for removing from cloud :)

**Comment by neox387** [comment]
test after next pr .. ^^

in settings enable:
Download RSS feeds & Delay in hours:

**Comment by buntin7419** [comment]
I'll have a play, Thanks al lot.

**Comment by buntin7419** [comment]
Had a quick play before going out this morning.
Are the RSS folders supposed to automatically download as I haven't managed to get this going.

I've enabled RSS downloads in Premiumizer settings. 
Added a show to Premiumize.me Rss
The show is added to Premiumize.me (but has no category name) so won't auto download.
Files download as expected if i manually select a category.

So basically no category is being assigned to files added via Rss
Would it be possible that Premiumier assigns the categaory of files to the Rss folder name it's in?
The Rss folder name can then be added to the Categories section of Premiumizer for auto download.

Or am I doing it wrong?
